lib/linguist/languages.yml

```diff
@@ -4856,13 +4856,20 @@ Volt:
   codemirror_mime_type: text/x-d
   language_id: 390
 Vue:
-  type: markup
+  type: programming
   color: "#2c3e50"
   extensions:
   - ".vue"
-  tm_scope: text.html.vue
+  tm_scope: source.vue
   ace_mode: html
+  codemirror_mode: vue
```

Comment:
```
  1) Failure:
TestLanguage#test_codemirror_modes_present [/home/travis/build/github/linguist/test/test_language.rb:424]:
#<Linguist::Language name=Vue> missing CodeMirror MIME mode
```

... is happening because you're missing a `codemirror_mime_type: <blah>` entry right about here.